src/main/java/it/gov/pagopa/rtd/ms/rtdmsdecrypter/service/IDecrypter.java

```diff
@@ -0,0 +1,12 @@
+package it.gov.pagopa.rtd.ms.rtdmsdecrypter.service;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+
+import it.gov.pagopa.rtd.ms.rtdmsdecrypter.model.BlobApplicationAware;
+
+public interface IDecrypter {
+  BlobApplicationAware decrypt(BlobApplicationAware fileName);
```

Comment:
I think the parameter should be named 'blob' as in the implementation class: fileName is a bit misleading given the variable type.

Reply:
I agree, I modified the name only in the implementation class